doc/users/whats_new/2015-01-19_cursor_pixel_data.rst

```diff
@@ -1,6 +1,22 @@
 Allow Artists to Display Pixel Data in Cursor
 ---------------------------------------------
 
-Adds `get_pixel_data` and `format_pixel_data` methods to artists
+Adds `get_cursor_data` and `format_cursor_data` methods to artists
 which can be used to add zdata to the cursor display
 in the status bar.  Also adds an implementation for Images.
+
+Added an property attribute ``mouseover`` and rcParam
```

Comment:
I was a bit confused by this, I think...  This PR is really just an optimization around a new feature that hasn't been released yet, right?

Then maybe instead of "Added an attribute..." say "This feature requires that the `axes.mouseover` attribute is set to `True`..." etc.

Barring that, I'm :+1: on this PR.